applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -397,6 +412,10 @@ prepare_context(Context, AccountId, AccountDb) ->
                                  ,{fun cb_context:set_account_id/2, AccountId}
                                 ]).
 
+-spec validate_patch(api_binary(), cb_context:context()) -> cb_context:context().
+validate_patch(AccountId, Context) ->
+    crossbar_doc:patch_and_validate(AccountId, Context, fun validate_request/2).
```

Comment:
I personally don't mind this, since we break out of `validate_account` to more specifically-named functions in general.